config/samples/devops.kubesphere.io_v1alpha3_pipeline.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: devops.kubesphere.io/v1alpha3
 kind: Pipeline
 metadata:
+  namespace: defaultg58cw
```

Comment:
I prefer to not have a namespace here. Users can give a new namespace when using `kubectl`.